lib/pure/cgi.nim

```diff
@@ -84,23 +84,17 @@ proc getEncodedData(allowedMethods: set[RequestMethod]): string =
 iterator decodeData*(data: string): tuple[key, value: TaintedString] =
   ## Reads and decodes CGI data and yields the (name, value) pairs the
   ## data consists of.
```

Comment:
`decodeData` should document `nimLegacyParseQueryStrict` option too.

> And even with `-d:nimLegacyParseQueryStrict`, it never raises `CgiError` now. It actually raises `UriParseError`.

ref https://github.com/nim-lang/Nim/pull/19308#issuecomment-1003737210